lib/reek/context/method_context.rb

```diff
@@ -85,6 +85,17 @@ def full_comment
         ''
       end
 
+      def class_method_block?
+        return false unless @parent_exp
+
+        result = @parent_exp.each_node(:send).select do |node|
+          node_args = *node
+          node_args.include?(:class_methods)
+        end
```

Comment:
Is there a way to `pattern match` the `s(:send, nil, class_methods)` directly?

Reply:
You can check `node.name == :class_methods`